src/INA138.cpp

```diff
@@ -0,0 +1,22 @@
+#include "INA138.hpp"
+
+namespace LVSS {
+
+INA138::INA138(IO::ADC& adc0) : ADC(adc0) {
+}
+
+uint32_t INA138::readCurrent(IO::ADC& adc0) {
+    //Gets voltage from adc
+    uint32_t voltage = static_cast<uint32_t>(adc0.read() * 1000);
```

Comment:
Gets voltage as a number from 0-4095
```suggestion
    uint32_t adcCounts = adc0.readRaw();
```